example/basic_robot_control.py

```diff
@@ -0,0 +1,67 @@
+import time
+
+import numpy as np
+
+from home_robot.motion.stretch import HelloStretch
+from home_robot_hw.client import StretchClient
+
+if __name__ == '__main__':
+    robot = StretchClient()
+    model = HelloStretch()
+
+    # Acquire camera observations
+    imgs = robot.camera.get_images()
+
+    # Get camera pose
+    camera_pose = robot.camera.get_pose()
+
+    # Move camera
+    robot.camera.set_pan_tilt(pan=np.pi / 4, tilt=-np.pi / 3)
+
+    # Switch to navigation mode
+    robot.switch_to_navigation_mode()
+    assert robot.in_navigation_mode()
+
+    # Get base pose
+    xyt = robot.nav.get_pose()
+
+    # Command robot velocities
+    robot.nav.set_velocity(v=0.1, w=0.0)
+    time.sleep(1)
+    robot.nav.set_velocity(v=0.0, w=0.0)
+
+    # Command the robot to navigate to a waypoint
+    xyt_goal = [0.25, 0.25, -np.pi / 2]
+    robot.nav.navigate_to(xyt_goal, blocking=True)
+
+    # Switch to manipulation mode
+    robot.switch_to_manipulation_mode()
+    assert robot.in_manipulation_mode()
+
+    # Get ee pose
+    pos, quat = robot.manip.get_ee_pose(relative=True)
+
+    # Command the robot arm 1
+    q_desired = np.random.randn(6)
+    pos_desired = np.array([-0.10281811, -0.7189281, 0.71703106])
+    quat_desired = np.array([-0.7079143, 0.12421559, 0.1409881, -0.68084526])
+
+    robot.manip.set_joint_positions(q_desired)
+    robot.manip.set_ee_pose(pos_desired, quat_desired)
+
+    # Command the robot arm 2
+    q = model.compute_ik(pos_desired, quat_desired)
```

Comment:
@cpaxton Any comments on the original design of the separate kinematic model?
I did notice that it is in spirit similar to command builders in other robot control libraries (such as the [RobotCommandBuilder](https://dev.bostondynamics.com/docs/python/understanding_spot_programming#commanding-the-robot) in the Spot SDK), maybe it's worth looking into the merits it provides.

Reply:
My original design had the kinematics object be var in the client actually. but i do think it's useful to have it separable - so it can appear in different types of clients, in simulators, in motion planners, etc.

Reply:
We can have it as a variable in the client, while also allowing it to be instantiated separately from `home_robot.motion.stretch`.
We can also create a `client.get_model()` method for easy access to the model instance being used by the client.